### ISSUE TITLE
Newly acquired items no longer show when connecting to an ongoing game

### ISSUE THREAD

**Issue by thislooksfun** [ArchipelagoMW-HollowKnight/Archipelago.HollowKnight#220]
**Describe the bug**
Previously when connecting to an ongoing Archipelago multiworld all the items other players gave you since the last time you played. But the current version doesn't do that anymore. It still displays items when I get them during the game, but I miss seeing all the items I got since last I connected.

**To Reproduce**
1. Create an Archipelago game
2. Connect to it
3. Save and quit
4. Get given an item (can just cheat it in with `/send`)
5. Reconnect to the game
6. The item will show up in RecentItemsDisplay, if installed, but not in the bottom left of the screen

**Expected behavior**
All the received items should show on-screen when connecting.

**Screenshots**
N/A

**Additional context**
I unfortunately do not remember the last version it worked in, but it's been a while (probably over 6 months) since I last did an HK Arch run, so it could be anywhere in that range...

**Comment by BadMagic100** [comment]
The behavior listed was an intentional change, especially when receiving many items it can keep the item queue in the corner blocked for a long time. I figured that generally in cases where it's relevant, players would use the tracker to see what items they had gotten offline.

Potentially I could consider this a feature request but would want to know more about the use case

**Comment by thislooksfun** [comment]
Yeah, it can stay blocked for a while for sure, but when using `RecentItemsDisplay` as well that isn't really a problem. Plus items don't get lost, just delayed.

My group likes to do long (multi-day) randomizers, and we often play asynchronously. That leads to getting a lot of items while I am offline. We found it fun to see all the items we got scroll by one by one. It made it a lot easier (for me at least) to keep track of all the important items I received, and it was a fun group activity to watch each other receive all the new items. With that feature removed it's now much harder for me to keep track of the new items available to me (or even be aware of them, without proactively looking).

**Comment by BadMagic100** [comment]
I may consider this as an option. It seems to me that the value of the corner display, especially if recent items display is not used, is severely reduced if blocked for several minutes. Plus going looking proactively, either at the tracker or at the inventory will generally be much more efficient. My inclination is that I probably wouldn't want to maintain this, but am not certain
